deepquest/__main__.py

```diff
@@ -67,7 +69,10 @@ def changes2dict(args):
                     changes[k] = v
                 else:
                     try:
-                        changes[k] = ast.literal_eval(v)
+                        if k == 'GPU_ID':
+                            changes[k] = str(v)
+                        else:
+                            changes[k] = ast.literal_eval(v)
```

Comment:
What prevent `ast.literal_eval()` from being used for GPU_ID as well?

Reply:
> This may have to be a temporary fix, but I realised that if GPU_ID is set
to something like `2,3` (ie with a comma) then `ast.literal_eval()` turns
it into a tuple, which causes problems later down the line.

If it is temporary, what would be the other solutions?

Reply:
I'm not sure, perhaps @drj11 might have a suggestion.

Reply:
Not really, other than to perhaps reverse the presumption of using `ast.literal_eval`.

`ast.literal_eval` is useful if you want something like a list or a complex number in your config values, but maybe we only need that for a few things, and we should use it only where we need it, instead of everywhere except `GPU_ID`?